static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -140,8 +143,10 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
   }
 
   getTitle() {
-    const {orgId} = this.props.params;
-    return routeTitleGen(t('Sentry Integration Details'), orgId, false);
+    const {app} = this.state;
+    const integrationAction = app ? 'Edit' : 'Create';
+    const integrationType = this.isInternal ? 'Internal' : 'Public';
+    return t('%s %s Integration', integrationAction, integrationType);
```

Comment:
I think `tct` is cleaner for this.
```suggestion
    return tct('[action] [type] Integration', { action: integrationAction, type: integrationType });
```

Reply:
Addressed! `getTitle` was throwing an error returning a component since it's declared in `asyncView.tsx` as returning a string so I renamed the method and converted to `tct`, thanks!